src/pages/hackathon/cypherpunk.tsx

```
@@ -182,9 +182,13 @@ function HeroMini({
   const [countdownDate, setCountdownDate] = useState<Date>(
     dayjs.utc(START_DATE).toDate(),
   );
-  const [status, setStatus] = useState<'Start In' | 'Close In' | 'Closed'>(
-    'Start In',
-  );
+  const [status, setStatus] = useState<'Start In' | 'Close In' | 'Closed'>(() =>
```

ESLint

[error] Insert `⏎····`

(prettier/prettier)